packages/react-composites/src/composites/CallWithChatComposite/components/MoreDrawer.tsx

```diff
@@ -170,6 +179,30 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
       onItemClick: props.onPeopleButtonClicked
     });
   }
+  /* @conditional-compile-remove(control-bar-button-injection) */
+  const customDrawerButtons = useMemo(
+    () =>
+      generateCustomCallWithChatDrawerButtons(
+        onFetchCustomButtonPropsTrampoline(drawerSelectionOptions !== false ? drawerSelectionOptions : undefined),
+        drawerSelectionOptions !== false ? drawerSelectionOptions?.displayType : undefined
+      ),
+    [drawerSelectionOptions]
+  );
+
+  /* @conditional-compile-remove(control-bar-button-injection) */
+  customDrawerButtons['overflow']?.props.children.forEach((element) => {
+    drawerMenuItems.push(element);
+  });
+  /* @conditional-compile-remove(control-bar-button-injection) */
+  customDrawerButtons['primary']?.props.children
+    .slice(CUSTOM_BUTTON_OPTIONS.MAX_PRIMARY_MOBILE_CUSTOM_BUTTONS)
```

Comment:
should this slice less if a user has hidden some buttons?

Reply:
For now, the goal is to provide clear and simplistic boundaries on how many custom buttons are injectable. 
We can then further explore the responsiveness of this design once we receive customer feedback.